.github/workflows/replace_links.yml

```diff
@@ -0,0 +1,34 @@
+name: Replace links with uids in copy branch
+
+on:
+  push:
+    branches: [ main ]
+    
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+    - name: Commit changes
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        $ThisCommit = "${{ github.sha }}"
+        git checkout ⭐Docs
+        git reset --hard $ThisCommit
+        pip install ./sync-script
+        replace_links https://raw.githubusercontent.com/geduardo/quantum-docs-pr/adding-file-script/articles/user-guide/language/uid-map.csv ./Specifications/Language/
```

Comment:
Fairly minor, but it may work better to check out the `quantum-docs-pr` repo itself rather than relying on `raw.githubusercontent.com`.

Reply:
Yeah, I considered that, but since it's only a single file and there are different working repos that will store the file in the process, I considered that the link is easier to avoid headaches with the authentication...

Also, I think I would need to clone the full repo and it would take longer for the script to run, right?